boards.txt

```diff
@@ -758,6 +758,54 @@ GenF4.menu.upload_method.bmpMethod=BMP (Black Magic Probe)
 GenF4.menu.upload_method.bmpMethod.upload.protocol=gdb_bmp
 GenF4.menu.upload_method.bmpMethod.upload.tool=bmp_upload
 
+###############################
+Vake403d.name=VAkE 403d
+
+Vake403d.build.vid=0x1EAF
+Vake403d.build.pid=0x0004
+Vake403d.vid.0=0x1EAF
+Vake403d.pid.0=0x0004
+
+Vake403d.build.core=arduino
+Vake403d.build.board=Vake403
+Vake403d.build.extra_flags=-D{build.product_line} {build.enable_usb} {build.xSerial}
+
+# Vake F446VE
+# Support: Serial1 (USART1 on PA10, PA9)
+# Default SPI: SPI
+Vake403d.menu.pnum.VAKE_F446VE=Vake F446VE
```

Comment:
Should I change "Vake F446VE" to "VAkE" ?  VAkE is infact the name of the PCB, VAke403d the name of the test printer for the PCB and since it's based on the F446...it started it's live a "Vake F446VE"